components/Search/results/ResidentsTable.tsx

```diff
@@ -28,7 +28,7 @@ const ResultTable = ({
 }: {
   records: LegacyResident[];
 }): React.ReactElement => (
-  <table className="govuk-table">
+  <table className="govuk-table" data-testid="residents-table">
```

Comment:
Good question – I accidentally introduced a subtle bug in our Cypress test, because I wrote a test that searched for a user, then asserted their ID was visible. 

The test passed (because the ID was 7, and that was visible), but it was visible _elsewhere_ on the page, not inside the table.

So I introduced this `data-testid` attribute so my Cypress test could assert that the ID was found _inside the table_, not just anywhere on the page.